blocks/post-teaser-list/post-teaser-list.js

```diff
@@ -1,4 +1,5 @@
-import { getRelatedPosts } from '../../scripts/scripts.js';
```

Comment:
Can we reserve some space for these cards? They may cause a CLS issue for real users on slow networks because, when scrolling down, the footer loads first and then these cards are added above it.

Reply:
open an issue and we'l track that there. I thought about wrapping it in an intersection observer to sidestep any issue.